curriculum/challenges/english/25-front-end-development/workshop-cat-photo-app/6698417821794837f14ca275.md

```diff
@@ -25,7 +25,7 @@ Your `p` element should have a nested anchor (`a`) element with the text `cat ph
 
 ```js
 const anchor = $('p > a');
-assert(
+assert.isTrue(
```

Comment:
please here use two asserts, one that checks if `anchor`has at least one element (assert.isNotEmpty), and then one that check the inner text of the first one (`assert.equal`)